openwisp_users/admin.py

```diff
@@ -256,6 +278,20 @@ def get_fields(self, request, obj=None):
             fields.insert(0, 'uuid')
         return fields
 
+    def has_change_permission(self, request, obj=None):
+        """
+        Allow operator change an organization only if
+        he is an admin of that organization
```

Comment:
use `they` instead of `he`, see here: https://en.wikipedia.org/wiki/Singular_they